client/app/dbaas/logs/detail/offer/logs-offer.less

```diff
@@ -0,0 +1,38 @@
+.cui-box {
```

Comment:
if you feel like this will be used elsewhere, extact it as a component.
Plus, do we really want a fixed with of 293px and a fixed height of 222px?

Reply:
I did create a component at the beginning but dont think it will be used anywhere else. So for now it will be a template. Will move it out as a component if i see any reusability